airflow/utils/db.py

```diff
@@ -308,6 +308,18 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id="ftp_default",
+            conn_type="ftp",
+            host="localhost",
```

Comment:
Added an ftp_default connection, as it previously did not exist.

Reply:
Even though the unit tests no longer use a local FTP Server, I still think that having this FTP Default connection would be useful, as without this, defining the following piece of code will throw an error indicating that the "connection with id 'ftp_default' does not exist": `hook = FTPHook()`